src/modules/navigation/components/PersonalInfoBlade/PersonalInfoBlade.module.scss

```
@@ -43,5 +43,9 @@ $component: "c-personal-info-blade";
 
 	&__send-button {
 		margin-bottom: $spacer-sm;
+
+		div {
+			font-weight: 600;
```

use scss variables where possible:
```suggestion
			font-weight: $font-weight-bold;
```